discoveryengine/list_documents_sample.py

```diff
@@ -44,5 +42,7 @@ def list_documents_sample(
     for result in response:
         print(result)
 
+    return response
```

Comment:
Consider returning a relevant object within the response that matches the name of the method. Is there a list that comes back as part of the response you can return?

Reply:
It's hard to get the results this way because it returns a Pager object.

I don't know if it's possible to put a type annotation for because it's not exposed as a type in the client library (but I could be wrong)